src/Aquarius.Client/TimeSeries/Client/ServiceModels/Provisioning.cs

```diff
@@ -180,6 +188,7 @@ public class GetInterpolationTypes
     public interface IFileUploadRequest
     {
         IHttpFile File { get; set; }
+        // HACK from generate_code_from_live_endpoint.sh // bool IsFileRequired { get; set; }
```

Comment:
@MartinBenn-AI FYI - The recent change to the `IFileUploadRequest` which added the `IsFileRequired` property exposed a ServiceStack code-generator bug that broke the SDK build.

You didn't do anything wrong, and I've hacked the SDK code-generation step to automatically fix it, but I thought I'd make you aware of it.